packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -98,19 +98,21 @@ class GlowingOverscrollIndicator extends StatefulWidget {
   _GlowingOverscrollIndicatorState createState() => new _GlowingOverscrollIndicatorState();
 
   @override
-  void debugFillDescription(List<String> description) {
-    super.debugFillDescription(description);
-    description.add('$axisDirection');
+  void debugFillProperties(List<DiagnosticsNode> description) {
+    super.debugFillProperties(description);
+    description.add(new EnumProperty<AxisDirection>('axisDirection', axisDirection));
+    String showMessage;
     if (showLeading && showTrailing) {
-      description.add('show: both sides');
+      showMessage = 'both sides';
     } else if (showLeading) {
-      description.add('show: leading side only');
+      showMessage = 'leading side only';
     } else if (showTrailing) {
-      description.add('show: trailing side only');
+      showMessage = 'trailing side only';
     } else {
-      description.add('show: neither side (!)');
+      showMessage = 'neither side (!)';
     }
-    description.add('$color');
+    description.add(new StringProperty('show', showMessage));
```

Comment:
(But maybe what i expect is bad, I don't feel strongly about this. It just seems weird to have "fake" properties.)